src/app.css

```diff
@@ -0,0 +1,51 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html {
+    background: #dae3e7
+}
+
+ul {
+    padding-left: 40px;
+}
+
+.centered {
+    width: 100%;
+    max-width: 1170px;
+    margin: 0 auto;
+}
+
+.container {
+    display: grid;
+    grid-template-rows: 240px 1fr 43px;
+}
+
+.header {
+    background: #f5f5f5;
+    border-top: 10px solid rgb(119, 132, 146);
```

Comment:
any particular reason you are using RGB here and hex in other places?

As an aside: some light preprocessing to this file might be a good idea - just so that you can use variables for colors and stuff.